src/main/java/digital/slovensko/autogram/core/SaveFileTarget.java

```diff
@@ -0,0 +1,29 @@
+package digital.slovensko.autogram.core;
+
+import java.io.File;
+import java.util.List;
+
+public class SaveFileTarget {
```

Comment:
Toto sa niekde používa? Nepodarilo sa mi nájsť.

Reply:
odstranene - to som zacal pisat nieco velmi podobne ako TargetPath